client-datastore-cassandra/src/main/java/com/telecominfraproject/wlan/client/datastore/cassandra/ClientSessionDAO.java

```diff
@@ -403,10 +431,29 @@ public ClientSession deleteSession(int customerId, long equipmentId, MacAddress
             strb.append(") ");
 
             query = strb.toString() + query;
-            query_head = "select macAddress from client_session_by_location where ";
+            query_head = "select macAddress from client_session_by_location where customerId = ? ";
```

Comment:
Did you test this? you added customerId to the query for the table that does not have it in a primary key, and did not provide the bind variable for it. I'm not seeing how this would work.